DO NOT PULL 0911 network auth updates TEAM REVIEW

maxwellfunk opened the pull request:
@JBPayne007 
@id2win 
@0Vanessa0 

Please disregard this pull request we are working on a mechanism to collaborate on a change that shows commit deltas.

@dproudGSA 
@rsherwood-gsa 

you should see the commits in this pull to see what has changed from the original

maxwellfunk committed:
Update scl-windows.md

Update top notice

maxwellfunk committed:
Update scl-windows.md

refine opening notice.

maxwellfunk committed:
Update scl-windows.md

update revocation information

maxwellfunk committed:
Update scl-windows.md

DC certificate updates for SID

maxwellfunk committed:
Update scl-windows.md

Added a note on mutual authentication in step 3

maxwellfunk committed:
Update scl-windows.md

syntax correction

maxwellfunk committed:
Update scl-windows.md

syntax fix

maxwellfunk committed:
Update scl-windows.md

updates to identifier strengths and tuple mappings

maxwellfunk committed:
Update scl-windows.md

typo correction

maxwellfunk committed:
Update scl-windows.md

spelling correction

maxwellfunk committed:
Update scl-windows.md

add specific group policy address for the triple mapping

maxwellfunk committed:
Update scl-windows.md

syntax update

maxwellfunk committed:
Update scl-windows.md

fix syntax

maxwellfunk committed:
Update scl-windows.md

fix syntax

maxwellfunk committed:
Update scl-windows.md

fix spelling errors and test punctuation.

maxwellfunk committed:
Update scl-windows.md

update step 5 to include reference to group policy for triple mapping

maxwellfunk committed:
Update scl-windows.md

fix formatting

maxwellfunk committed:
Update scl-windows.md

updating for sample tuple mappings

maxwellfunk committed:
Update scl-windows.md

Update to include specific examples of policy tuples

maxwellfunk committed:
Update scl-windows.md

typo corrections

maxwellfunk committed:
Update scl-windows.md

streamline

dproudGSA commented on _implement/scl-windows.md:
```diff
@@ -33,21 +33,15 @@ subnav:
+    <h4 class="usa-alert__heading">September 2024 - Update to Microsoft Network Authentication Issue</h4>
     <p class="usa-alert__text">
-      The Microsoft KB mentioned above is updated. Note that the "disabled" mode retirement is still targeted at 2/14/23. CISA encourages any agency still reliant on "disabled" mode to move to "compatibility mode" by following the <a class="usa-link usa-link--external" href="https://www.cisa.gov/guidance-applying-june-microsoft-patch" target="_blank" rel="noopener noreferrer">CISA Guidance</a> as ...
+      As of September 10th 2024, Microsoft has released a solution for Active Directory network authentication issues resulting from the May 2022 patches that impacted some PIV network authenications. The September patch applies to Windows Server 2019 and later and includes a mechanism for support of what are considered "weak" identifiers asserted by PIV authentication certificates (e.g., UPN or  ...
```
```suggestion
      As of September 10th 2024, Microsoft has released a solution for Active Directory network authentication issues resulting from the May 2022 patches that impacted some PIV network authentications. The September patch applies to Windows Server 2019 and later and includes a mechanism for support of "weak" identifiers asserted by PIV authentication certificates (e.g., UPN or X509IssuerSubject altsecid) and mapped to AD user accounts.  AD administrators now have the ability to add registry keys that include what is being termed a "Triple Mapping" or "Policy Tuple" that allows the domain controller to determine if an authentication certificate is issued from a trusted Certification Authority (CA) and if it asserts an acceptable policy OID before defining acceptable identifiers, which are then considered "strong."  You can read more about these AD changes in the following <a class="usa-link usa-link--external" href="https://techcommunity.microsoft.com/t5/public-sector-blog/enable-strong-name-based-mapping-in-government-scenarios/ba-p/4240402" target="_blank" rel="noopener noreferrer">Microsoft Public Sector Blog</a>.  Full enforcement mode for use of strong identifiers is still planned to go into effect on February 11, 2025 and compatibility mode will be fully retired on September 10th, 2025.  See Step 4 below regarding Account Linking for further details.
```

dproudGSA reviewed (commented):
Just a few minor edits.

maxwellfunk commented:
Reopening, please DO NOT PULL as we are making some team modifications and verbiage corrections per CISA

maxwellfunk committed:
Update _implement/scl-windows.md

Reads fine to me, commiting.  We are going to have to update this per CISA's terminology update to move away from strong and weak regardless.

Co-authored-by: Diana Proud-Madruga <118391820+dproudGSA@users.noreply.github.com>

rsherwood-gsa commented on _implement/scl-windows.md:
```diff
@@ -33,21 +33,15 @@ subnav:
+    <h4 class="usa-alert__heading">September 2024 - Update to Microsoft Network Authentication Issue</h4>
     <p class="usa-alert__text">
-      The Microsoft KB mentioned above is updated. Note that the "disabled" mode retirement is still targeted at 2/14/23. CISA encourages any agency still reliant on "disabled" mode to move to "compatibility mode" by following the <a class="usa-link usa-link--external" href="https://www.cisa.gov/guidance-applying-june-microsoft-patch" target="_blank" rel="noopener noreferrer">CISA Guidance</a> as ...
+      As of September 10th 2024, Microsoft has released a solution for Active Directory network authentication issues resulting from the May 2022 patches that impacted some PIV network authentications. The September patch applies to Windows Server 2019 and later and includes a mechanism for support of "weak" identifiers asserted by PIV authentication certificates (e.g., UPN or X509IssuerSubject a ...
```
Even though "weak" is in quotes, I still think it will worry people unnecessarily. I suggest the following replacement text: 

... The September patch applies to Windows Server 2019 and later and includes a mechanism to support mapping of identifiers asserted by PIV authentication certificates (e.g., UPN or X509IssuerSubject altsecid) to AD user accounts.  AD administrators can define a "Triple Mapping" or "Policy Tuple" of Certification Authority (CA), Policy OID and acceptable identifiers. Identifiers included in the policy tuple are then considered "strong" for purposes of AD authentication. ...

maxwellfunk replied:
I have since updated to "supported" "supported with policy" and "deprecated" per CISA

maxwellfunk committed:
Update scl-windows.md

updates to remove reference to weak vs strong microsoft identifier categorization

rsherwood-gsa commented on _implement/scl-windows.md:
```diff
@@ -63,8 +57,8 @@ Check the following items **before** reviewing these network guides and lessons
-1. Domain Controllers are Microsoft 2012 or newer.
-1. User workstations **are joined** to your network and are Windows 8 or Windows 10-based.
+1. Domain Controllers are running Microsoft Windows Server 2019 or newer.
+1. User workstations **are joined** to your network and are Windows 8 or newer.
```
Windows 8 is no longer supported. We should specify Windows 10 or later.

maxwellfunk replied:
agreed

maxwellfunk committed:
Update scl-windows.md

update verbiage

maxwellfunk committed:
Update scl-windows.md

include prohibition on DC certificates being part of a policy tuple

maxwellfunk committed:
Update scl-windows.md

syntax correction

maxwellfunk committed:
Update scl-windows.md

syntax correction

maxwellfunk committed:
Update scl-windows.md

maxwellfunk committed:
Update scl-windows.md

removal of offending alert

maxwellfunk committed:
Update scl-windows.md

fix an alert

maxwellfunk committed:
Update scl-windows.md

syntax

maxwellfunk committed:
Update scl-windows.md

syntax

maxwellfunk committed:
Update scl-windows.md

maxwellfunk committed:
Update scl-windows.md

update windows 8 to 10

maxwellfunk committed:
Update scl-windows.md

update to include general ADCS device restrictions.

maxwellfunk committed:
Update scl-windows.md

correct spelling and remove redundant alert

maxwellfunk committed:
Update scl-windows.md

spelling correction.
